neurodsp/sim/info.py

```diff
@@ -23,6 +21,8 @@ def get_sim_funcs(module_name):
         A dictionary containing the available sim functions from the requested sub-module.
     """
 
+    from neurodsp.sim import periodic, aperiodic, transients, combined, cycles
```

Comment:
Why were these imports moved to inside the func?

Reply:
Circular imports! All hell breaks loose if they are at the top (it breaks). 

This function is programmatically quirky (not ideal, in best practices sense), but it's efficient at what it does, so I sorta like it.